plugins/source/aws/codegen/recipes/iam.go

```diff
@@ -460,15 +440,10 @@ func IAMResources() []*Resource {
 						Type:    schema.TypeString,
 						Options: schema.ColumnCreationOptions{PrimaryKey: true},
 					},
-					{
-						Name:     "tags",
-						Type:     schema.TypeJSON,
-						Resolver: `client.ResolveTags`,
-					},
 					{
 						Name:     "user_tags",
 						Type:     schema.TypeJSON,
-						Resolver: `client.ResolveTags`,
+						Resolver: `client.ResolveTags`, // FIXME this is wrong, add path
```

Comment:
This needs to be `client.ResolveTagField("User.Tags")` or something like that but I'm not sure about the syntax...

Reply:
Tested: User tags are not exposed in the `User` struct from `ListVirtualMFADevices`, so removing this column.